README.md

```diff
@@ -8,7 +8,7 @@
 
 ## About
 
-Coveo CLI is a command line interface to interact with the Coveo platform. It allows you to quickly create a [Coveo Headless](https://docs.coveo.com/headless)-powered search page for your Coveo organization, relying on [Angular](https://angular.io), [React](https://reactjs.org/) or [Vue.js](https://vuejs.org/).
+Coveo CLI is a command line interface to interact with the Coveo platform. With this tool, you are able to create [Coveo Headless](https://docs.coveo.com/headless)-powered search pages, push data to your organization, automate resource deployments, and create your own test organizations.
```

Comment:
For now I think it's nice, prolly outta find a more generic thingy later down the road; pretty sure we'll forget to keep that updated :D